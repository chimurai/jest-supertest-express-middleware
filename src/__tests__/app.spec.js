const request = require('supertest')
const app = require('../app')

describe('GET /user', () => {
  it('respond with json', async () => {
    const response = await request(app)
      .get('/user')
      .set('Accept', 'application/json')
      .expect('Content-Type', /json/)
      .expect(200)

    expect(response.body).toMatchSnapshot()
  })
})
