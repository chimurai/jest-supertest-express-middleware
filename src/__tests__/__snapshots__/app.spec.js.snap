// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /user respond with json 1`] = `
Object {
  "name": "john",
}
`;
